media-sound/zyn-fusion/zyn-fusion-3.0.5.ebuild

```diff
@@ -0,0 +1,116 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+USE_RUBY="ruby25 ruby26 ruby27"
+PYTHON_COMPAT=( python{2_7,3_6,3_7,3_8,3_9} )
```

Comment:
Is `PYTHON_REQ_USE='threads(+)'` no longer required for waf?

Reply:
Huh, hadn't heard of this one. I guess it is. I'll add it.